src/c++/perf_analyzer/genai-perf/genai_perf/parser.py

```diff
@@ -345,6 +345,13 @@ def _add_endpoint_args(parser):
 def _add_output_args(parser):
     output_group = parser.add_argument_group("Output")
 
+    output_group.add_argument(
+        "--generate-graphs",
```

Comment:
I think we should stop mixing `graph` and `plot` words in our codebase for consistency. IMO, we should stick to `plot` since our class names are `PlotManager`, `BoxPlot`, etc.
```suggestion
        "--generate-plot",
```